pkg/reconciler/revision/resources/queue_test.go

```diff
@@ -71,12 +72,14 @@ var (
 		InitialScale:          1,
 		AllowZeroInitialScale: false,
 	}
-	deploymentConfig deployment.Config
-	logConfig        logging.Config
-	obsConfig        metrics.ObservabilityConfig
-	traceConfig      tracingconfig.Config
-	defaults, _      = apicfg.NewDefaultsConfigFromMap(nil)
-	revCfg           = config.Config{
+	deploymentConfig = deployment.Config{
+		ProgressDeadline: 5678 * time.Second,
```

Comment:
heh, yeah, it's a bit of a pain tho because we need 5678 * time.Second here and some other places, but plain ol' 5678 in the TimeoutSeconds fields (because it's not a duration), and "1h34m38s" in the `-probe-period` flag, so it's either 3 constants or the tests have to duplicate the code a bit :(. I vote we clean it up in a follow-up :D